app/src/components/adminConsole/currentCup/CurrentCupHeader.tsx

```diff
@@ -1,13 +1,29 @@
 import React from "react";
 import CreateCupButton from "./CreateCupButton";
+import {
+    SectionHeader,
+    MiniElleImg,
+    NewCupButton,
+} from "../AdminConsoleStyles";
+//import Elle from "../../../images/Elle.svg";
+
 const CurrentCupHeader: React.FunctionComponent = (props: {
     isCup: Boolean;
-    isOpen: Boolean;
+    updateIsCup: () => void;
 }) => {
     return (
-        <div>
-            <h5>Current SnapCups</h5>
-            <CreateCupButton isCup={props.isCup} isOpen={props.isOpen} />
+        <div className="container">
+            <div id="currentCupHeader" className="row">
+                <div className="col-md-3">
+                    <SectionHeader>Current SnapCup</SectionHeader>
+                </div>
+                <div className="col-md-9">
```

Comment:
What should these do at other breakpoints?

Reply:
I think nothing? There's only two things in the row, so they're separate below medium and together above